src/steamship/experimental/transports/chat.py

```diff
@@ -6,6 +6,9 @@
 
 
 class ChatMessage(Block):
+
+    who: Optional[str] = "bot"
```

Comment:
Was this supposed to be in this pr?

Reply:
It was yes, without it the front-end thinks the message was written by the user.